documentation/docs/requirements/system-overview.md

```diff
@@ -29,16 +30,18 @@ Devs and Dragons is a multiplayer web-based application that provides a gamified
 - Grading does not start until all members have taken a turn
 - Ensures that proper logic is being used to solve
   - A problem expecting an output of "10" will not accept "return 10" as a correct input
-- GPT rates successful answers from 1 to 3 stars
-  - 1 being extremely inefficient
-  - 2 being decently efficient
-  - 3 being the most efficient
+- GPT rates answers from 1 to 5 stars, based on a 10-point grading scale
+  - **1 star**: Grade 0–3/10 (Extremely inefficient; irrelevant or incorrect)
+  - **2 stars**: Grade 4–5/10 (Inefficient; partially helpful but unclear or incomplete)
+  - **3 stars**: Grade 6–7/10 (Efficient; clear, relevant, and complete)
+  - **4 stars**: Grade 8–9/10 (Highly efficient; exceeds expectations)
+  - **5 stars**: Grade 10/10 (Exceptional; goes above and beyond)
 
 ### Quest Creation
 All users can create a quest by filling out a form that specifies the difficulty, number of problems, computer science related topic(s), enemy, and background.
 
 ### Starting a Quest
-User must be in a party of two or more people in order to start a quest made by the user. Guild parties can start a quest that is assigned to their guild. Multiple guild parties can start and complete the same quest.
+In multiplayer mode, the user must be in a party of two or more people in order to start a quest made by the user. The user who created the quest has the ability to provide a unique code for others to join the quest.
 
 ## Conceptual Design
 The frontend of Devs and Dragons will be developed using React.js to build UI components and manage the application state. The backend will be powered by Flask, handling user authentication, API requests, and integrating with the OpenAI GPT model for generating and grading coding challenges. MongoDB will be the database, storing user data, quest configurations, and coding submissions. React Game Kit will develop the game component using its tools to create and manage animations and game mechanics.
```

Comment:
We are no longer using React Game Kit, the game is being developed using ReactJS and JS. Include that we are using web sockets to handle the multiplayer connection.